python/tests/test_tree_stats.py

```diff
@@ -3310,19 +3305,22 @@ def get_tree_sequence(self):
                               mutation_rate=2, random_seed=1)
         return ts
 
+    def identity_f(self, ts):
```

Comment:
Ah, I thought I commented on this already, seems to have gotten lost. We're redefining these functions a bunch of times --- I think it'd be better to just define them at the top-level stating what they're for. We don't use any state from the class anyway.